zproject/backends.py

```diff
@@ -917,17 +917,28 @@ def social_associate_user_helper(backend: BaseAuth, return_data: Dict[str, Any],
 
         if not chosen_email:
             avatars = {}  # Dict[str, str]
+            existing_account_emails = []
             for email in verified_emails:
                 existing_account = common_get_active_user(email, realm, {})
                 if existing_account is not None:
+                    existing_account_emails.append(email)
                     avatars[email] = avatar_url(existing_account)
-
-            return render(backend.strategy.request, 'zerver/social_auth_select_email.html', context = {
-                'primary_email': verified_emails[0],
-                'verified_non_primary_emails': verified_emails[1:],
-                'backend': 'github',
-                'avatar_urls': avatars,
+            if(len(existing_account_emails) > 1):
```

Comment:
It looks like you have some linting issues. Use `./tools/lint` on your dev env to see the issues. 

The problem with this line here is the lack of space:
```
Jan 04 19:27:03 py        | Put a space between statements like if, else, etc. and (. at zproject/backends.py line 926:
Jan 04 19:27:03 py        |             if(len(existing_account_emails) > 1):

```

Reply:
Thanks @vinitS101 I've fixed the linting issue.